docs/assets/index-DmJnWrCa.js

```
@@ -0,0 +1,84 @@
+import { B as w } from './index-BFNIYgDS.js';
+const f = async (a) => {
+    try {
+      await window.kasware.requestAccounts(),
+        (await window.kasware.getNetwork()) !== a && (await window.kasware.switchNetwork(a));
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)